src/routes/__test__/Router.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, it, vi } from 'vitest'
+import { render, screen } from '@testing-library/react'
+import CssBaseline from '@mui/material/CssBaseline'
+import CustomThemeProvider from '@/theme/CustomThemeProvider'
+import Router from '../Router'
+import '@/i18n/config'
+
+// Mock all the route components
```

Comment:
This seems like a good practice to mock the various parts. Would it be possible to DRY up the mocks in to an array plus a function?